arrays/arrays.c

```diff
@@ -21,11 +21,14 @@ typedef struct Array {
  *****/
 Array *create_array (int capacity) {
   // Allocate memory for the Array struct
-
+  Array *newarray = malloc(sizeof(Array));
   // Set initial values for capacity and count
-
+  newarray->capacity = capacity;
+  newarray->count = 0;
   // Allocate memory for elements
+  newarray->elements = malloc(sizeof(char) * capacity);
```

Comment:
You should be allocating space for the char* rather than char in your `sizeof(char)` argument. It also might be more convenient if you use calloc instead of malloc for this part.